packages/patrol/android/src/main/kotlin/pl/leancode/patrol/BrowserstackPatrolJUnitRunner.java

```diff
@@ -18,7 +18,7 @@ public PatrolAppServiceClient createAppServiceClient() {
 
         try {
             client.listDartTests();
-        } catch (StatusRuntimeException ex) {
+        } catch (PatrolAppServiceClientException ex) {//TODO
```

Comment:
why TODO here?

Reply:
I am not sure if it is this error or another exception from khttp will be correct in this case. So I asked you about a project where we use Browserstack, where we can verify this place :)

Reply:
OK, let's merge it as is into `develop` and I'll check if it works on BrowserStack.